### DISCUSSION TITLE
Rosetta support for Apple Silicon

### DISCUSSION THREAD

**Discussion by planetmarshall** [containers/podman#21241] [Ideas]
As of Macos 13.0, Apple supports using Rosetta in Linux VMs (I believe Docker Desktop now has support for this)

See https://developer.apple.com/documentation/virtualization/running_intel_binaries_in_linux_vms_with_rosetta

It would be great to see this feature in Podman, I did do a search but didn't see any existing work underway.

**Comment by rhatdan** [comment]
Working on it.  Podman 5.0 will default to apple hypervisor and have support for this. Should ship in February.

**Reply by herrernst** [reply]
PR: https://github.com/containers/podman/pull/21670
